openvmm/openvmm_resources/src/lib.rs

```diff
@@ -17,6 +17,8 @@ vm_resource::register_static_resolvers! {
     tpm::resolver::TpmDeviceResolver,
     #[cfg(guest_arch = "x86_64")]
     serial_16550::resolver::Serial16550Resolver,
+    #[cfg(guest_arch = "x86_64")]
```

Comment:
How come?

Reply:
Oh, because it uses IO ports.

Reply:
No other VMM supports the debugcon on non-x86 platforms, so while we could certainly be the first, I didn't think it was too important a mmio mode at the moment.

Reply:
Makes sense. I suppose modern firmwares use real serial ports (OVMF notwithstanding?)

Reply:
I mean, even SeaBIOS technically supports sending debug logs via a real serial port with a compile-time tweak (e.g: for use in baremetal / coreboot scenarios). But this debugcon device is pretty useful for super minimal, no-frills, ad-hoc debugging for inside the guest,